public/scss/components/OrderSummaryBox.module.scss

```
@@ -425,6 +414,46 @@
         }
     }
 
+    &_voucherValidList,
+    &_voucherInvalidList
+    {
+        @include flex(column, center, center);
+        position: relative;
+        width: 100%;
+        padding: calc(8px + 0.8vh) calc(8px + 0.8vw) calc(26px + 2.6vh);
+
+        & > * 
+        {
+        width: 100%;
+        max-width: 1170px; 
+        margin: 0;

```

```suggestion
            width: 100%;
            max-width: 1170px; 
            margin: 0;

```

indent-nya kurang nih nyonya